polaris-core/src/main/java/io/polaris/core/persistence/resolver/PolarisResolutionManifest.java

```diff
@@ -403,7 +403,8 @@ public PolarisResolvedPathWrapper getResolvedTopLevelEntity(
     if (resolvedCacheEntry == null) {
       return null;
     }
-    return new PolarisResolvedPathWrapper(
-        List.of(getResolvedRootContainerEntity(), new ResolvedPolarisEntity(resolvedCacheEntry)));
+
+    ResolvedPolarisEntity resolvedRootContainerEntity = getResolvedRootContainerEntity();
+    return resolvedRootContainerEntity == null ? new PolarisResolvedPathWrapper(List.of(new ResolvedPolarisEntity(resolvedCacheEntry))):new PolarisResolvedPathWrapper(List.of(resolvedRootContainerEntity, new ResolvedPolarisEntity(resolvedCacheEntry))) ;
```

Comment:
This line looks too long. How about wrapping like below? 
```java
    return resolvedRootContainerEntity == null
            ? new PolarisResolvedPathWrapper(List.of(new ResolvedPolarisEntity(resolvedCacheEntry)))
            : new PolarisResolvedPathWrapper(List.of(resolvedRootContainerEntity, new ResolvedPolarisEntity(resolvedCacheEntry))) ;
```

Reply:
I was hoping build will fail with spotless check.